omod/src/main/webapp/resources/htmlforms/HMIS-ACP-018-HTSClientCard.xml

```diff
@@ -543,15 +543,15 @@
                                 <div class="card-body">
                                     <div class="row">
                                         <div class="col-md-4">
-                                            <label>First HIV test (has never tested before):</label>
+                                            <label>First HIV test (has never tested before)</label>
                                             <obs id="165180" conceptId="165180" answerConceptIds="90003,90004" answerLabels="1 - Yes,2 - No" style="radio" class="horizontal"/>
                                         </div>
                                         <div class="col-md-4">
-                                            <label>Last HIV Visit Date:</label>
+                                            <label>Last hiv visit date</label>
```

Comment:
Use HIV instead of hiv for consistence purposes, since the rest of the labels have HIV